cmd/krew/cmd/root.go

```diff
@@ -141,8 +141,12 @@ func preRun(cmd *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
-	if !isMigrated && cmd.Use != "receipts-upgrade" {
-		fmt.Fprintln(os.Stderr, "You need to perform a migration to continue using krew.\nPlease run `kubectl krew system receipts-upgrade`")
+	if !isMigrated {
+		fmt.Fprintln(os.Stderr, `This version of Krew is not supported anymore. Please manually migrate:
+1. Backup plugin list: kubectl krew list > backup.txt
```

Comment:
So here's the problem: `krew list` will not work, because the krew we're running here is a new one, but it is faced with an old index. Instead of listing plugins, `krew list` will simply produce this error message and bail out. That's the reason why I was advocating for a unified migration.

The instructions here cannot rely on any `krew` command and must rely on `bash` (BSD flavor, so that OSX works) and PowerShell or other Windows instructions.

Reply:
Ah good catch, I can change this. I can come up with something for bash, but I don't really have any experience in windows :sweat_smile:. I'll try to figure out what it would look like there, but do you or @ahmetb know what the windows equivalent of `ls -d $KREW_HOME/store/ > plugins.tmp`/`kubectl krew install < plugins.tmp` would be?

Reply:
Nice catch.
I'd say don't bother with this and just talk about:

1. uninstall...
2. reinstall...
3. install the plugins you used

(don't mention backups).